README.md

```diff
@@ -60,6 +60,41 @@ $$
 function_info = client.create_function(sql_function_body=sql_body)
 ```
 
+#### Create a UC python function
+
+The client offers an API that simplifies the process of directly creating a python function to be registered as a UC function.
+
+To get the optimal results for GenAI tool definitions, it is highly recommended to:
+
+- Provide a full Python docstring in Google docstring format to your function definition. This API extracts the information about arguments so that GenAI tools
```

Comment:
```suggestion
- Provide a full Python docstring in [Google docstring format](URL) to your function definition. This API extracts the information about arguments so that GenAI tools
```

Reply:
Ah, good idea! Since we're using mkdocs, markdown comment syntax should just work in the generated API docs :)